### PR TITLE
GH-200 Add option to exclude Creative mode players from combat

### PR DESCRIPTION
**Pull request by CitralFlo**
Additionally refactor config variable name

### CODE REVIEW

**Comment by Jakubk15** [comment]
@coderabbitai review

File: eternalcombat-plugin/src/main/java/com/eternalcode/combat/fight/controller/FightTagController.java [hunk 1]
```diff
@@ -54,11 +55,13 @@ void onEntityDamageByEntity(EntityDamageByEntityEvent event) {
...
 
-        if (attacker.isOp() && this.config.settings.excludeAdminFromCombat) {
+        if (attacker.isOp() && this.config.settings.excludeOpFromCombat ||
+            attacker.getGameMode().equals(GameMode.CREATIVE) && this.config.settings.excludeCreativeFromCombat ) {
             return;
         }
 
-        if (attackedPlayerByPerson.isOp() && this.config.settings.excludeAdminFromCombat) {
+        if (attackedPlayerByPerson.isOp() && this.config.settings.excludeOpFromCombat ||
+            attackedPlayerByPerson.getGameMode().equals(GameMode.CREATIVE) && this.config.settings.excludeCreativeFromCombat) {
```
Comment (Rollczi) [inline eternalcombat-plugin/src/main/java/com/eternalcode/combat/fight/controller/FightTagController.java:64]: Extract this to private method `canTag(Player): boolean`

Comment (CitralFlo) [reply eternalcombat-plugin/src/main/java/com/eternalcode/combat/fight/controller/FightTagController.java:64]: yessir

Comment (Rollczi) [reply eternalcombat-plugin/src/main/java/com/eternalcode/combat/fight/controller/FightTagController.java:64]: 😳

**Review by Rollczi — APPROVED** [review]

**Review by P1otrulla — APPROVED** [review]

**Pull request merged**